src/python/bot/fuzzers/honggfuzz/engine.py

```diff
@@ -109,7 +109,8 @@ class HonggfuzzEngine(engine.Engine):
   def name(self):
     return 'honggfuzz'
 
-  def prepare(self, corpus_dir, target_path, build_dir):
+  # FIXME: Add support for build_dir argument
```

Comment:
nit: remove this comment. this argument is optionally handled and doesn't make sense here.

Reply:
yes, sure.